CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+## Contributing
+Thank you for your interest in contributing to Codewind.
+
+#### Before you contribute
+- Contributions must follow the Eclipse organization specifications. Refer to the [Git commit instructions](https://www.eclipse.org/projects/handbook/#resources-commit).
+
+#### Adding contributions
+- See the [Codewind GitHub Workflows](https://wiki.eclipse.org/Codewind_GitHub_Workflows) wiki.
+
+## Raising an issue
+
+#### Before you raise an issue
+- Search the [list of issues](https://github.com/eclipse/codewind-eclipse/issues) to see if your issue has already been raised.
```

Comment:
This link to point to the codewind issues link https://github.com/eclipse/codewind/issues since we use a single repo to keep track on all the issues.